install/kubernetes/cilium/values.yaml

```diff
@@ -889,7 +889,7 @@ hubble:
         - sourceLabels:
             - __meta_kubernetes_pod_node_name
           targetLabel: node
-          replacement: ${1}
```

Comment:
Something probably broken on my local setup as this differs from what CI comes up with. Will try to figure out what's up.

Reply:
`envsubst -no-digit` fixes it. Seems like something changed in newer versions of `envsubst` and CI was probably using a very old version. I'll look into this issue in a separate PR and see what we can do about it.